src/pages/contact-us/_offices.js

```diff
@@ -264,34 +306,38 @@ export const Offices = () => {
                                 </MapIconWrapper>
                                 <Text target="_blank" color="black-3">
                                     <Localize
-                                        translate_text="1902, Jumeirah Business Center 1, <0 /> JLT Cluster G, Dubai, UAE."
+                                        translate_text="1902, Jumeirah Business Center 1,<0/>JLT Cluster G, Dubai, UAE."
                                         components={[<br key={0} />]}
                                     />
                                 </Text>
                             </AddressContainer>
                             <StyledText>
                                 <PhoneIcon />
-                                +9714 399 0404
+                                <ClickToCall href="tel:+97143990404">+971 4 399 0404</ClickToCall>
                             </StyledText>
                         </Content>
                         <ImageWrapper>
-                            <Image
-                                img_name="map-dubai-office.png"
-                                alt={localize('Dubai Office')}
-                                width="49rem"
-                                height="100%"
-                            />
+                            <MapContainer>
+                                <Image
```

Comment:
Image component will generate 5 images. please use QueryImage which only generate 2 component instead, we want to reduce the image usage as much as possible 🙏

Reply:
we can do in different PR instead